### PR TITLE
SPCTR-2298 Wp-VIP-Go fixes

### PR DESCRIPTION
**Pull request by adi3890**
### Description
Fixing Wordpress-Vip-Go issues

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-beta-updates.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ private function get_beta_version() {
 			if ( false === $beta_version ) {
 				$beta_version = 'false';
 
-				$response = wp_remote_get( 'https://plugins.svn.wordpress.org/ultimate-addons-for-gutenberg/trunk/readme.txt' );
+				$response = wp_remote_get( 'https://plugins.svn.wordpress.org/ultimate-addons-for-gutenberg/trunk/readme.txt' ); // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.wp_remote_get_wp_remote_get
```
Comment (sandeshjangam) [inline classes/class-uagb-beta-updates.php:83]: WordPressVIPMinimum.Functions.RestrictedFunctions.wp_remote_get_wp_remote_get

Ignore this rule globally in PHPCS file

File: lib/gutenberg-templates/classes/class-ast-block-templates.php [hunk 2]
```diff
@@ -306,7 +306,7 @@ public function activate_plugin() {
 
 			wp_clean_plugins_cache();
 
-			$plugin_init = ( isset( $_POST['init'] ) ) ? esc_attr( $_POST['init'] ) : ''; // phpcs:ignore
+			$plugin_init = ( isset( $_POST['init'] ) ) ? esc_attr( $_POST['init'] ) : '';
```
Comment (sandeshjangam) [inline lib/gutenberg-templates/classes/class-ast-block-templates.php:309]: esc_attr( $_POST['init'] ) 

sanitize_text_field will be correct here

Comment (adi3890) [reply lib/gutenberg-templates/classes/class-ast-block-templates.php:309]: yes, will update accordingly.

File: classes/class-uagb-beta-updates.php (hunk 1 above)
Comment (adi3890) [reply classes/class-uagb-beta-updates.php]: done

File: lib/gutenberg-templates/classes/class-ast-block-templates.php (hunk 2 above)
Comment (adi3890) [reply lib/gutenberg-templates/classes/class-ast-block-templates.php:309]: done

File: phpcs.xml.dist [hunk 3]
```diff
@@ -9,6 +9,16 @@
...
 	<rule ref="WordPress-VIP-Go" />
+
+	<rule ref="WordPress-VIP-Go">
+		<exclude name="WordPressVIPMinimum.Functions.RestrictedFunctions.wp_mail_wp_mail" />
+		<exclude name="WordPressVIPMinimum.Variables.RestrictedVariables.cache_constraints___SERVER__REMOTE_ADDR__" />
+		<exclude name="WordPressVIPMinimum.Variables.RestrictedVariables.cache_constraints___SERVER__HTTP_USER_AGENT__" />
+		<exclude name="WordPressVIPMinimum.Functions.RestrictedFunctions.get_intermediate_image_sizes_get_intermediate_image_sizes" />
+		<exclude name="WordPressVIPMinimum.Variables.ServerVariables.UserControlledHeaders" />
+		<exclude name="WordPressVIPMinimum.Functions.RestrictedFunctions.wp_remote_get_wp_remote_get" />
+		<exclude name="WordPressVIPMinimum.Hooks.AlwaysReturnInFilter.MissingReturnStatement" />
```
Comment (sandeshjangam) [inline phpcs.xml.dist:20]: Ignore this inline as this is exceptional in some cases. We follow this approach. always return in filter.

Comment (adi3890) [reply phpcs.xml.dist]: done

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**